src/psyclone/domain/gocean/transformations/kern_trans.py

```diff
@@ -37,6 +37,8 @@
 PSyclone kernel-layer-specific PSyIR which uses specialised classes.
 
 '''
+import re
+
 from psyclone.domain.gocean.kernel import GOceanKernelMetadata, GOceanContainer
 from psyclone.psyGen import Transformation
 from psyclone.psyir.nodes import Container, ScopingNode, FileContainer
```

Comment:
I've belatedly realised that `KernTrans` is not a very descriptive name! Perhaps `RaiseRoutine2KernTrans` or something?

Reply:
RaisePSyIR2GOceanKernelTrans